tests/helpers.py

```diff
@@ -18,10 +18,6 @@ class MockComponentA(Observer):
     def name(self) -> str:
         return self._name
 
-    @property
```

Comment:
We actually use the defaults now. It is also very weird we use this class for component and observer tests.